src/util/helpers.ts

```diff
@@ -33,18 +33,18 @@ export const limitLength = (value: string, len: number) => value.substr(0, len);
 
 export const isAddressValid = (address: string) => /^[a-km-zA-HJ-NP-Z1-9]{26,35}$/.test(address);
 
-export const parseSenderAddresses = sender => {
-  if (!sender.length) {
+export const parseAddresses = address => {
```

Comment:
This should probably be called `addresses`. And potentially a better function name would be `serializeAddresses` or `stringifyAddresses` or something.

Reply:
Parsing usually goes in the other direction.